runtime-modules/common/src/lib.rs

```diff
@@ -4,3 +4,27 @@
 pub mod constraints;
 pub mod currency;
 pub mod origin_validator;
+
+use codec::{Decode, Encode};
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+/// Defines time in both block number and substrate time abstraction.
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Clone, Encode, Decode, PartialEq, Eq, Debug, Default)]
+pub struct BlockAndTime<BlockNumber, Moment> {
+    /// Defines chain block
+    pub block: BlockNumber,
+
+    /// Defines time
+    pub time: Moment,
+}
+
+/// Gathers current block and time information for the runtime
```

Comment:
Can we add a comment here that explains if this function is used inside a `config()` at genesis the timestamp will be `0` because the timestamp is actually produced by validators.

Reply:
I copy your comments to the code comments.